cg/cli/clean.py

```diff
@@ -154,3 +155,26 @@ def mipauto(context: click.Context, before_str: str, yes: bool = False):
                 f"{family_id}: sample_info file not found, please mark the analysis as deleted in "
                 f"the analysis table in trailblazer."
             )
+
+
+@clean.command()
+@click.option("-c", "--case-id", type=str)
+@click.option("-n", "--number-of-conversions", type=int)
+@click.option("-d", "--dry-run", is_flag=True)
+def compress(context, case_id, number_of_conversions, dry_run):
```

Comment:
Maybe this command should be separated into more than one depending on "responsibility", compress files depending on a cg-case and compress files depending on what is found in scout?